test/source/tests/gmail.ts

```diff
@@ -322,15 +345,16 @@ export const defineGmailTests = (testVariant: TestVariant, testWithBrowser: Test
       )
     );
 
-    test(
+    // draft-sensitive test
+    test.serial(
```

Comment:
It'll be run serially anyway. I can leave this `serial` as precaution or delete it

Reply:
Is there another config somewhere for running these 4 tests serially?

Reply:
all gmail live tests are run serially: --concurrency=1
https://github.com/FlowCrypt/flowcrypt-browser/blob/cbb2aff53d2f894e41e7c22371103cb546eb172b/package.json#L84

Reply:
We found that `--concurrency=1` is applied only to number of concurrent test files, but there can be more than 1 concurrent test - https://github.com/FlowCrypt/enterprise-server/issues/4807. So it'll be better to leave `test.serial`